src/app/modules/ix-table/components/ix-table-body/cells/ix-cell-state-button/ix-cell-state-button.component.ts

```diff
@@ -88,11 +90,11 @@ export class IxCellStateButtonComponent<T> extends ColumnComponent<T> {
           this.errorHandler.showErrorModal(error);
         });
         dialogRef.componentInstance.aborted.pipe(untilDestroyed(this)).subscribe(() => {
+          this.snackbar.success(this.translate.instant('Task Aborted'));
           dialogRef.close();
-          this.dialogService.info(this.translate.instant('Task Aborted'), '');
         });
       } else if (state.state === JobState.Hold) {
-        this.dialogService.info(this.translate.instant('Task is on hold'), state.reason);
+        this.snackbar.success(this.translate.instant('Task is on hold'));
```

Comment:
I don't think it's appropriate to change it here. 
1. It hides the reason. 
2. It's not a success message. 
3. Snackbars are for dismissible messages after user actions, where user has already done something. In here, user could open job dialog, go away and not know what happened.